### DISCUSSION TITLE
Smokeping-users appears abandoned

### DISCUSSION THREAD

**Discussion by fuzzyeric** [oetiker/SmokePing#403] [Q&A] [answered]
At https://lists.oetiker.ch/ , the mailing list smokeping-users is given as the current discussion mechanism for SmokePing work.  Subscribe requests produce

> The recipient server did not accept our requests to connect. For more information, go to https://support.google.com/mail/answer/7720 [[lists.oetiker.ch](http://lists.oetiker.ch/) 2a02:aa08:e000:1e00::a8: timed out] [[lists.oetiker.ch](http://lists.oetiker.ch/) [46.140.183.220](http://46.140.183.220/): FAILED_PRECONDITION: connect error (111): Connection refused]

https://lists.oetiker.ch/ should be updated to point wherever is currently the place for such discussion.

**Comment by oetiker** [comment, marked as answer]
Hi @fuzzyeric discussions have been moved here!
